server/akka-http-server/src/test/scala/sttp/tapir/server/akkahttp/AkkaHttpCreateServerTest.scala

```diff
@@ -43,6 +53,34 @@ class AkkaHttpCreateServerTest extends TestSuite {
               basicRequest.get(uri"http://localhost:$port/api/test/directive").send(backend).map(_.body shouldBe Right("ok"))
             }
             .unsafeRunSync()
+        },
+        Test("Send and receive SSE") {
+          implicit val ec = scala.concurrent.ExecutionContext.Implicits.global
+          val e = endpoint.get
+            .in("sse")
+            .out(serverSentEventBody)
+            .serverLogic[Future](_ => {
+              Source(List(sse1, sse2)).asRight[Unit].unit(new FutureMonad())
+            })
+          val route = AkkaHttpServerInterpreter.toRoute(e)
+          interpreter
+            .server(NonEmptyList.of(route))
+            .use { port =>
+              IO.fromFuture {
+                IO(
+                  basicRequest
+                    .get(uri"http://localhost:$port/sse")
+                    .response(
+                      asStreamUnsafe(AkkaStreams).mapRight(stream =>
+                        AkkaHttpServerInterpreter.parseBytesToSSE(stream).runFold(List.empty[ServerSentEvent])((acc, sse) => acc :+ sse)
+                      )
+                    )
+                    .send(AkkaHttpBackend())
```

Comment:
I think we can use `backend` without creating a new one (and creating a new `ActorSystem`)

Reply:
We probably can't because the already defined `backend` has `Fs2Stream` streaming capabilities, not the `AkkaStreams`.

Reply:
then we need to close the backend after usage, or better, re-use the exisiting actor system when creating it so that a new one isn't created at all

Reply:
I have changed it to `AkkaHttpBackend.usingActorSystem` to reuse the `ActorSystem` and properly clean up resources :)